src/PyNvCodec/CMakeLists.txt

```diff
@@ -43,21 +44,39 @@ fetchcontent_declare(
 )
 fetchcontent_makeavailable(pybind11)
 
-pybind11_add_module(_PyNvCodec MODULE 
-	src/PyBufferUploader.cpp
-	src/PyCudaBufferDownloader.cpp
-	src/PyFFMpegDecoder.cpp
-	src/PyFFMpegDemuxer.cpp
-	src/PyFrameUploader.cpp
-	src/PyNvCodec.cpp
-	src/PyNvDecoder.cpp
-	src/PyNvEncoder.cpp
-	src/PySurface.cpp
-	src/PySurfaceConverter.cpp
-	src/PySurfaceDownloader.cpp
-	src/PySurfaceRemaper.cpp
-	src/PySurfaceResizer.cpp
-)
+if(BUILD_TEGRA)
+  add_definitions(-DTEGRA_BUILD)
+    pybind11_add_module(_PyNvCodec MODULE 
+      src/PyBufferUploader.cpp
+      src/PyCudaBufferDownloader.cpp
+      src/PyFFMpegDecoder.cpp
+      src/PyFFMpegDemuxer.cpp
+      src/PyFrameUploader.cpp
+      src/PyNvCodec.cpp
+      src/PySurface.cpp
+      src/PySurfaceConverter.cpp
+      src/PySurfaceDownloader.cpp
+      src/PySurfaceRemaper.cpp
+      src/PySurfaceResizer.cpp
```

Comment:
This would be nicer using a CMake variable outside the if as you would have to change 2 locations in the future otherwise.

Reply:
You're probably right, it can be substituted with `list(APPEND ...)`
Could you please do it when merging PR?